module/config.mjs

```diff
@@ -2719,6 +2719,31 @@ preLocalize("attackClassifications", { key: "label" });
 
 /* -------------------------------------------- */
 
+/**
+ * Attack modes available for weapons.
+ * @enum {string}
+ */
+DND5E.attackModes = Object.seal({
+  oneHanded: {
+    label: "DND5E.ATTACK.Mode.OneHanded"
+  },
+  twoHanded: {
+    label: "DND5E.ATTACK.Mode.TwoHanded"
+  },
+  offhand: {
+    label: "DND5E.ATTACK.Mode.Offhand"
+  },
+  thrown: {
+    label: "DND5E.ATTACK.Mode.Thrown"
+  },
+  "thrown-offhand": {
```

Comment:
Any strong feelings regarding renaming this `thrownOffhand`?

Reply:
It complicates some code elsewhere, because as it is we can simply check `mode.endsWith("offhand")` to match both `offhand` and `thrown-offhand` modes.